berkeley-mobile/Map/MapMarkerDetailView.swift

```diff
@@ -168,10 +173,16 @@ class MapMarkerDetailView: UIView {
     }
     
     @objc func closeView() {
+        
         marker = nil
-        delegate?.didCloseMarkerDetailView(self)
+        UIView.transition(with: self,
+                          duration: 1.0,
+                                 options: [.transitionFlipFromBottom],
+                                 animations: {
+                                 },
+                                 completion: nil)
```

Comment:
the spacing/indent is a bit off here, you cant hit ctrl+i to auto indent or adjust yourself